Language/Functions/Analog IO/analogRead.adoc

```diff
@@ -1,102 +1,102 @@
 ---
-title: analogRead()
-categories: [ "Functions" ]
-subCategories: [ "Analog I/O" ]
+शीर्षक: analogRead()
+श्रेणियाँ: [ "कार्यों" ]
+उपश्रेणियों: [ "एनालॉग आई/ओ" ]
 ---
 
 = analogRead()
 
-// OVERVIEW SECTION STARTS
-[#overview]
+// अवलोकन अनुभाग शुरू होता है
+[#अवलोकन]
```

Comment:
This asciidoc markup generates anchor links in the web page. For example, this URL takes you to the Overview section of the analogRead() reference page:
https://www.arduino.cc/reference/hi/language/functions/analog-io/analogread/#अवलोकन

My question is whether it's better to leave these standardized so that you can switch between one language to another by only changing the language code in the URL. For example
- https://www.arduino.cc/reference/hi/language/functions/analog-io/analogread/#overview
- https://www.arduino.cc/reference/en/language/functions/analog-io/analogread/#overview

I'm not sure what's best. If you are going to translate them then I think it's best for you to translate all of them. You currently only translated the `overview` anchors and left the others (`howtouse`, `see_also`) untranslated.

Reply:
>I'll do the necessary changes as mentioned by you and submit the request with more translations.

Are you planning to do that in this pull request or in a separate pull request?

Reply:
@per1234 I planned to submit the modifications along with more translations in a separate pull request.